### PR TITLE
Add Play Your Way from SpaceDock

### PR DESCRIPTION
**Pull request by Space-Duck**
This pull request was automatically generated by SpaceDock on behalf of Berlin88, to add [Play Your Way](https://spacedock.info/mod/1101/Play%20Your%20Way) to CKAN.

Mod details:
    name = /mod/1101/Play%20Your%20Way
    author = Berlin88
    abstract = No contracts. Get rewarded for doing what you want!
    license = Public Domain
    Homepage = http://forum.kerbalspaceprogram.com/index.php?/topic/152877-play-your-way-121/#comment-2871751
    description = 
# Play Your Way
**v1.0 Initial Release**

Play Your Way removes ALL contracts, and frees up your imagination to explore the way you see fit. 
Thats not all though! Play Your Way also rewards you with REP and pays you FUNDS based on any science points you accumulate while you explore. 

Also, how much you get paid is completely up to you!

Want to get paid 1,000 FUNDS for every science point you collect? You got it! 500 REP for every science point you collect? Go for it!
Just make the necessary changes in the cfg.

So, basically, there's no contracts and you get paid whatever you want for doing whatever you want!

This can either make the game really hard or really easy, depending on how you set your cfg.

If you want contracts AND to still get rewarded from doing experiments, just delete the NoContracts.cfg from the PlayYourWay folder in your GameData.

ENJOY!

This is basically a continuation of Ippo's ScienceFunding but with a few tweaks. This mod keeps the same license as ScienceFunding.

License:
Public domain

Please direct questions about this pull request to [Berlin88](https://spacedock.info/profile/Berlin88).

### CODE REVIEW

**Comment by politas** [comment]
Should be fine, now

**Comment by Dazpoet** [comment]
Added a dependency on ModuleManager,
